### DISCUSSION TITLE
快速开始中python脚本的错误

### DISCUSSION THREAD

**Discussion by hqxwu** [PaddlePaddle/PaddleOCR#13573] [Q&A]
<img width="565" alt="1" src="https://github.com/user-attachments/assets/7ddb4864-b239-45ea-899e-4fffb7f4f7f2">
在运行官方提供的python脚本时报这个错误，调试发现错误发生在while str_count(txt) >= img_w // font_size - 4:这句代码，其中的txt为float

**Comment by SWHL** [comment]
运行的完整代码以及运行环境贴一下
